Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/ScreenData.purs

```diff
@@ -94,6 +94,14 @@ initData = {
       , index: 0
       , id: ""
       }
+    , tipViewState : {
+        primaryText : "Add a tip to find a ride quicker!"
+      , secondaryText : "It seems to be taking longer than usual."
+      , customerTipArray : ["₹10 🙂", "₹20 😄", "₹30 🤩"]
+      , customerTipArrayWithValues : [10, 20, 30]
```

Comment:
@aryanguptaSG customerTipArrayWithValues values can be [0,10, 20, 30] as there is not tip option

Reply:
Both Views are different . in this view there is no "no tip" option .